src/languages/bash.coffee

```diff
@@ -27,5 +27,9 @@ module.exports = {
       default: null
       minimum: 0
       description: "Indentation size/length"
+    use_tabs:
```

Comment:
We try to keep the option names the same as other languages. In this case, `indent_with_tabs` would be the appropriate option name which we already use.